eventkit_cloud/ui/templates/styles/Style.qgs

```diff
@@ -3,6 +3,8 @@
   <title></title>
   <layer-tree-group expanded="1" checked="Qt::Checked" name="">
     <customproperties/>
+    <layer-tree-group expanded="1" checked="Qt::Checked" name="Eventkit OSM Thematic">
```

Comment:
The layer name should be a variable.  The task that generates this file already uses the provider slug you can use that to query the name of the specific provider (or simply pass in the layer name to the task), in this case "OSM Data (Themes)".  This allows the user to change the name of the service and have it reflected in the template, or if we support multiple schemas allow this to work without needing to change it.